pkg/gitprovider/bitbucket.go

```diff
@@ -469,18 +472,18 @@ func (g *BitbucketGitProvider) GetDefaultBranch(staticContext *StaticGitContext)
 	}
 
 	for _, branch := range branches.Branches {
-		if branch.Type == "main" {
+		if branch.Name == "main" {
```

Comment:
This here won't work correctly since user might change default branch from `main` to something else. Please, revert this to the state it was in.
This is truly a bug from our side but it was fixed earlier today [here](https://github.com/daytonaio/daytona/pull/1057) which my colleagues will check and test tomorrow.
That PR we will merge first and you'll rebase this PR on top of that one once I notify you it's ready and this PR is ready as well.

Reply:
Ok!

Reply:
Alright then waiting for your further instructions!